src/python/pants/jvm/subsystems.py

```diff
@@ -22,12 +29,19 @@ def register_options(cls, register):
             default="adopt:1.11",
             advanced=True,
             help=(
-                "The JDK to use.\n\n"
-                " This string will be passed directly to Coursier's `--jvm` parameter."
-                " Run `cs java --available` to see a list of available JVM versions on your platform.\n\n"
-                " If the string 'system' is passed, Coursier's `--system-jvm` option will be used"
-                " instead, but note that this can lead to inconsistent behavior since the JVM version"
-                " will be whatever happens to be found first on the system's PATH."
+                "The JDK to use when building Pants' internal JVM support code and other "
+                "non-compiler tools."
```

Comment:
It's fine to move the help out to the subsystem, but this should likely reference it.